qiskit/scheduler/lowering.py

```diff
@@ -65,7 +70,9 @@ def lower_gates(circuit: QuantumCircuit, schedule_config: ScheduleConfig) -> Lis
     # convert the unit of durations from SI to dt before lowering
     circuit = convert_durations_to_dt(circuit, dt_in_sec=schedule_config.dt, inplace=False)
 
-    def get_measure_schedule(qubit_mem_slots: Dict[int, int]) -> CircuitPulseDef:
+    def get_measure_schedule(
+        qubit_mem_slots: Dict[int, int], backend: Optional[Union[BackendV1, BackendV2]] = None
```

Comment:
`get_measure_schedule` is a inner function and you can pass and use `backend`  without adding `backend`  as an argument.
```suggestion
    def get_measure_schedule(qubit_mem_slots: Dict[int, int]) -> CircuitPulseDef:
```